lsp/intellij-plugin/build.gradle.kts

```diff
@@ -0,0 +1,81 @@
+// Copyright 2000-2022 JetBrains s.r.o. and contributors. Use of this source code is governed by the Apache 2.0 license.
+
+plugins {
+  id("java")
+  id("org.jetbrains.kotlin.jvm") version "1.7.20-RC"
+  id("org.jetbrains.intellij") version "1.9.0"
+}
+
+group = "${Settings.groupId}.lsp.intellij-plugin"
+version = Settings.version
+
+repositories {
+  mavenCentral()
+  maven{
+    url = uri("https://jitpack.io")
+  }
+}
+
+// Include the generated files in the source set
+sourceSets {
+  main {
+    java {
+      srcDirs("src/main/gen")
+    }
+  }
+}
+
+dependencies {
+  implementation(project(":compiler:core"))
+  testImplementation("junit:junit:4.13.2")
```

Comment:
junit 4?

Reply:
Er zit nog geen test in ik verwijder de dep